lib/sidekiq_flow/front/assets/stylesheets/app.css

```diff
@@ -11,6 +11,7 @@ svg#workflow-graph {
   overflow: hidden;
   height: 70vh;
   padding-top: 10px;
+  padding-left: 0px !important;
```

Comment:
padding is in fact set/overwrite by graph.js `.css("padding-left",(getWidth(t)-getWidth(e))/2+"px")`
It is used to center the graph when page is ready, but I think padding is not the right attribute for that (should be a SVG translate function).

Reply:
updated PR